lib/index.ts

```diff
@@ -194,11 +198,29 @@ export class Compiler {
   private _compileParenthesizedTypeNode(node: ts.ParenthesizedTypeNode): string {
     return this.compileNode(node.type);
   }
+  private _compileImportDeclaration(node: ts.ImportDeclaration): string {
+    if (!this.options.traverseImports) {
+      return '';
+    }
+    const importedSym = this.checker.getSymbolAtLocation(node.moduleSpecifier);
+    if (importedSym && importedSym.declarations) {
+      return importedSym.declarations.map(declaration => this.compileNode(declaration)).join("");
```

Comment:
Will this end up calling _compileSourceFile? Could you add a comment to clarify this?

Reply:
yes, it does eventually call that which is why the topNode thing is important. I will make a comment to clarify that.